src/ansys/dpf/core/server_factory.py

```diff
@@ -602,13 +602,14 @@ def get_server_type_from_config(
         if config.protocol == CommunicationProtocols.gRPC and config.legacy:
             return LegacyGrpcServer
         elif config.protocol == CommunicationProtocols.gRPC and not config.legacy:
-            if ansys_path is None:
-                from ansys.dpf.core.misc import __ansys_version__
-
-                ansys_path = os.environ.get("AWP_ROOT" + str(__ansys_version__), None)
-            if ansys_path is not None:
-                sub_folders = os.path.join(ansys_path, _get_path_in_install())
-                os.environ["PATH"] += sub_folders
+            # if ansys_path is None:
```

Comment:
what was that for?

Reply:
This has been added when we switched to `ansys.dpf.gate`. Apparently adding the path to `GrpcClient.dll`. Not required today.